monitoring/README.md

```diff
@@ -4,38 +4,67 @@
 
 
 ### Description
-This example shows how to deploy monitoring tools (Prometheus, Grafana and Pyroscope) for KubeStellar control plane components - see architecture image above. These instructions can be used to deploy monitoring tools to any KubeStellar system. If you already have the monitoring tools installed in your environment, you can skip to `step-2`.
+This example shows how to deploy monitoring tools (Prometheus/Thanos, Grafana and Pyroscope) for KubeStellar control plane components - see architecture image above. These instructions can be used to deploy monitoring tools to any KubeStellar system. If you already have the monitoring tools installed in your environment, you can skip to `step-2`.
 
 #### 1. Install the monitoring tools:
 
 Starting from a local directory containing the git repo, do the following:
 
+a) Deploy the monitoring tools in the hosting cluster (e.g., `kind-kubeflex`):
+
+```bash
+$ cd monitoring
+$ ./install-ks-monitoring.sh --cluster-context kind-kubeflex --set core
+```
```

Comment:
I see a "copy" icon for this displayed text, but doing so is useless because of the included command prompts (the dollar signs).

<img width="887" alt="image" src="https://github.com/user-attachments/assets/a767f46a-bfec-430f-a744-1d83101a891f">

Reply:
Github adds the copy item in the README by default, is it?

Reply:
Let me try an experiment here.

```shell
echo my command
```

```console
$ echo my command
my command
```

Reply:
I was hoping that using `console` rather than `shell` as the [info-string](https://github.github.com/gfm/#info-string) (based on the list found from https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) would produce different rendering, but experiment shows "nope".

Reply:
FYI, you _could_ use the style used in the rest of our documentation, where every fenced block is _either_ command input (with prompt omitted) or command output --- but not a mixture of both. That way, readers can simply use the copy button on the command input blocks.

Reply:
Done! Fixed